vector/src/main/java/im/vector/util/PreferencesManager.java

```diff
@@ -202,8 +202,6 @@
             SETTINGS_NOTIFICATION_RINGTONE_PREFERENCE_KEY,
             SETTINGS_NOTIFICATION_RINGTONE_SELECTION_PREFERENCE_KEY,
 
-            SETTINGS_SET_SYNC_TIMEOUT_PREFERENCE_KEY,
```

Comment:
Not sure, but didn't you tell me to be careful when removing preference for backward compatibility?

Reply:
Actually it's removed because it's already in the list. It was added twice by mistake